### PR TITLE
#373 Resolves immutable updatedAt column

### LINKED ISSUE #373: The updatedAt property of SlurFilter is immutable
Seeing this error in the logs:

```
HHH000502: The [updatedAt] property of the [com.sublinks.sublinksapi.slurfilter.entities.SlurFilter] entity was modified, but it won't be updated because the property is immutable.
```

### PR DESCRIPTION
**Pull request by jgrim**
Resolves #373

### CODE REVIEW

**Comment by jgrim** [comment]
@Pdzly I've thought more about this issue. I think we should remove the db triggers and let springboot manage updatedAt for the entities. Rather than removing the annotation I believe we should make the field mutable and allow springboot to handle it. I'd consider updating the annotations for update and create to be driven by the database timestamp like so:

`@UpdateTimestamp(source = SourceType.DB)` & `@CreationTimestamp(source = SourceType.DB)`

**Comment by Pdzly** [comment]
> @Pdzly I've thought more about this issue. I think we should remove the db triggers and let springboot manage updatedAt for the entities. Rather than removing the annotation I believe we should make the field mutable and allow springboot to handle it. I'd consider updating the annotations for update and create to be driven by the database timestamp like so:
> 
> `@UpdateTimestamp(source = SourceType.DB)` & `@CreationTimestamp(source = SourceType.DB)`

Ok

**Comment by Pdzly** [comment]
Fixed it

**Review by lazyguru — DISMISSED** [review]
lgtm

File: src/main/java/com/sublinks/sublinksapi/community/entities/Community.java [hunk 1]
```diff
@@ -110,12 +111,13 @@ public class Community implements Serializable {
...
 
-  @CreationTimestamp
+  @CreationTimestamp(source = SourceType.DB)
   @Column(updatable = false, nullable = false, name = "created_at")
   private Date createdAt;
 
-  @UpdateTimestamp
-  @Column(updatable = false, nullable = false, name = "updated_at")
+  
+  @UpdateTimestamp(source = SourceType .DB)
```
Comment (lazyguru) [inline src/main/java/com/sublinks/sublinksapi/community/entities/Community.java:119]: extra space here between `SourceType` and `.DB`

**Review by lazyguru — CHANGES_REQUESTED** [review]

**Review by lazyguru — APPROVED** [review]

**Pull request merged**